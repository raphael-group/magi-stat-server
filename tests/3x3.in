{
    "X": [
	"1", "2", "2", "3", "1", "1", "1", "2", "3", "1", "1", "2", "2"
    ],

    "Y": [
	"1", "1", "1", "1", "2", "2", "2", "2", "2", "3", "3", "3", "3"	 
    ]
}