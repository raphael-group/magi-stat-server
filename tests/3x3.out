{
    "request": "contingency", 
    "samplesRemoved": 0, 
    "stats": {
        "Chi square test, (1, 2, 3) on (1, 2, 3) membership": {
            "DOF": 4, 
            "Expected": [
                [
                    1.8462, 
                    2.3077, 
                    1.8462
                ], 
                [
                    1.5385, 
                    1.9231, 
                    1.5385
                ], 
                [
                    0.6154, 
                    0.7692, 
                    0.6154
                ]
            ], 
            "Valid (all cells > 5)": false, 
            "X": "1, 2, 3", 
            "Y": "1, 2, 3", 
            "chi2": 2.2533, 
            "p": 0.6893, 
            "report": {
                "html": "&chi;<sup>2</sup>(4) = 2.25333, <b>P-value</b> = 0.689278", 
                "tex": "$\\chi^2(4) = 2.2533, P = 0.6893$"
            }
        }
    }, 
    "table": [
        [
            "", 
            "1", 
            "2", 
            "3"
        ], 
        [
            "1", 
            1.0, 
            3.0, 
            2.0
        ], 
        [
            "2", 
            2.0, 
            1.0, 
            2.0
        ], 
        [
            "3", 
            1.0, 
            1.0, 
            0.0
        ]
    ]
}
